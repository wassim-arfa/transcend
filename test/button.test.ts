test("Snapshot sum", () => {
    // I'm too lazy to figure out what this should equal...
    expect(3492 + 2592).toMatchSnapshot();
  })