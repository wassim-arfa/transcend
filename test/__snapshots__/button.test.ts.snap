// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot sum 1`] = `6084`;
